### DISCUSSION TITLE
Better integration with Cypress cloud

### DISCUSSION THREAD

**Discussion by AvtandilGachechiladze** [FRSOURCE/cypress-plugin-visual-regression-diff#223] [Ideas]
Hi,

I'm currently utilizing Cypress Cloud to record test runs. When a test fails due to discrepancies between the base and actual screenshots, only the actual screenshot is visible in the cloud. It would be more beneficial to also view the differences between the two images.

**Comment by donwelsh** [comment]
Hi @FRSgit. I would also like better integration with Cypress Cloud and have the differences screenshot up on Cypress Cloud as well.

**Comment by FRSgit** [comment]
Hey!

Unfortunately, I've never worked with Cypress Cloud, so I have no idea on how to follow up with this request. Do you know any 3rd party integrations (not done by Cypress developers themselves) with Cypress Cloud? Is it even possible?
